HTML/mockup-minimo/style.css

```diff
@@ -0,0 +1,360 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+html {
+    /* 1rem = 10px  10px/16px = 0.625 = 62.5% */
+    font-size: 62.5%;
+}
+
+@media only screen and (max-width: 500px) {
+    html {
+        font-size: 60%;
+    }
+}
+
+body {
+    margin: 0 auto 0 auto;
+    width: 120rem;
+}
+
+@media only screen and (max-width: 1200px) {
+    body {
+        width: 90%;
+    }
+}
+
+/******** HEADER ********/
+
+header {
+    margin: 6.8rem auto 0 auto;
+    width: 88.5%;
+}
+
+.nav {
+    width: 100%;
+    display: flex;
+    justify-content: space-between;
+}
+
+.logo {
+    padding: 0;
+    text-transform: uppercase;
+    font-family: Inconsolata;
+    font-size: 3rem;
+    font-weight: normal;
+    line-height: 1.6;
+    letter-spacing: 0.6rem;
+    text-align: center;
+    color: #000000;
+}
+
+.menu {
+    width: 60%;
+    list-style-type: none;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+}
+
+@media only screen and (max-width: 800px) {
+    .menu {
+        display: none;
+    }
+}
+
+.menu__item {
+    padding: 0;
+    align-items: center;
```

Comment:
probably possible to delete this, it is property for flex container [flex properties](https://css-tricks.com/snippets/css/a-guide-to-flexbox/)